device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsReceiverLinkHandler.java

```diff
@@ -118,13 +135,13 @@ public void onLinkRemoteClose(Event event)
         Link link = event.getLink();
         if (link.getLocalState() == EndpointState.ACTIVE)
         {
-            log.debug("{} receiver link with link correlation id {} was closed remotely unexpectedly", getLinkInstanceType(), this.linkCorrelationId);
```

Comment:
should we use trace here instead... line 160 has similar info but that is a trace instead of debug.